### PR TITLE
Update README and versions for 21.04 branch

### PR DESCRIPTION
**Pull request by dzier**
FOR REVIEW ONLY.

David, please review closely. There were a lot of new files in the deploy/ folder that I updated. Please make sure they are correct and let me know if I need to revert anything.

### CODE REVIEW

File: build.py [hunk 1]
```diff
@@ -64,7 +64,7 @@
 # incorrectly load the other version of the openvino libraries.
 #
 TRITON_VERSION_MAP = {
-    '2.9.0dev': ('21.04dev', '21.03', '1.7.1', '2021.2.200', '2021.2.200')
+    '2.9.0': ('21.04', '21.03', '1.7.1', '2021.2.200', '2021.2.200')
```
Comment (deadeyegoodwin) [inline build.py:67]: '21.04', '21.04', ...

File: deploy/gke-marketplace-app/server-deployer/build_and_push.sh [hunk 2]
```diff
@@ -28,7 +28,7 @@ export REGISTRY=gcr.io/$(gcloud config get-value project | tr ':' '/')
 export APP_NAME=tritonserver
 export MAJOR_VERSION=2.8
 export MINOR_VERSION=2.8.6
-export NGC_VERSION=21.03-py3
+export NGC_VERSION=21.04-py3
```
Comment (deadeyegoodwin) [inline deploy/gke-marketplace-app/server-deployer/build_and_push.sh:31]: Don't update anything in gke-marketplace-app for now. @mengdong should we make these kind of updates for each release?

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

File: deploy/gke-marketplace-app/server-deployer/build_and_push.sh (hunk 2 above)
Comment (mengdong) [reply deploy/gke-marketplace-app/server-deployer/build_and_push.sh]: I can make an update for 21.04 tomorrow. There are a few other places that need to be updated.

Comment (mengdong) [reply deploy/gke-marketplace-app/server-deployer/build_and_push.sh]: and yes, please keep this one.

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request closed without merging**